.env

```diff
@@ -2,7 +2,7 @@ HTTPS = false
 HTTPS_CERT_PATH = server.cert
 HTTPS_KEY_PATH = server.key
 REACT_APP_ALT_DRUG = true
-REACT_APP_AUTH = http://localhost:8180
+REACT_APP_AUTH = http://localhost:8180/auth/realms
```

Comment:
Should the default be /realms without the /auth since that is what is needed for newer versions of key cloak?